src/trix/models/block.coffee

```diff
@@ -137,13 +143,13 @@ class Trix.Block extends Trix.Object
     @attributes[depth]
 
   canBeGroupedWith: (otherBlock, depth) ->
-    attributes = @attributes
     otherAttributes = otherBlock.getAttributes()
-    if attributes[depth] is otherAttributes[depth]
-      if attributes[depth] in ["bullet", "number"] and otherAttributes[depth + 1] not in ["bulletList", "numberList"]
-        false
-      else
-        true
+    otherAttribute = otherAttributes[depth]
+    attribute = @attributes[depth]
+
+    attribute is otherAttribute and
+      not (Trix.config.blockAttributes[attribute].group is false and
```

Comment:
Yeah, I just mean in general it seems like if we’re going to have helpers for querying attribute configuration, then we should have helpers for everything we need to query, so there’s not two different interfaces to the same data.

I haven’t looked at the specific call sites. Maybe you can go through them and see if you spot any patterns?